include/swift/AST/DiagnosticsSema.def

```diff
@@ -1015,6 +1011,13 @@ ERROR(cannot_pass_inout_arg_to_subscript,none,
       "'withUnsafeMutablePointer' to explicitly convert argument "
       "to a pointer", ())
 
+ERROR(incorrect_property_wrapper_reference,none,
+      "cannot convert value %0 of type %1 to expected type %2, use %select{wrapper| wrapped value}3 instead",
+      (Identifier, Type, Type, bool))
+ERROR(incorrect_property_wrapper_reference_member,none,
+      "referencing to member %0 requires %select{wrapper|wrapped value of type}1 %2",
+      (DeclName, bool, Type))
```

Comment:
One more requiest - could you please make it `(DescriptiveDeclKind, DeclName, bool, Type)` and change test a bit to "referencing %0 %1 requires %select{wrapper|wrapped value of type}2 %3"?

Reply:
Actually let's just drop `to` for now, we can look into adding decl kind separately.